libs/govrn-subgraph-client/src/lib/subgraph/queries/queries.graphql

```diff
@@ -51,26 +55,3 @@ query listContributions(
     ...contributionFields
   }
 }
-
-fragment guildFields on Guild {
```

Comment:
is this related to the PR?

Reply:
Yea, we need to add fields to the subgraph and this prevented us from adding those fields